src/Common/Http/oeHttpRequest.php

```
@@ -143,22 +149,22 @@ public function setParams($params)
         ]));
     }
 
-    public function usingBaseUri($baseUri)
+    public function usingBaseUri(string $baseUri)
     {
         $baseUri = str_ends_with((string) $baseUri, '/') ? $baseUri : $baseUri . '/';
         return $this->tap($this, fn($request): array => $this->options = array_merge($this->options, [
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)